CHANGELOG.md

```diff
@@ -1264,5 +1285,7 @@ These should not affect the functionality of the module.
 
 
 
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
+
 
-\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
\ No newline at end of file
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```

Comment:
is it intentional that this is in here twice?

Reply:
fixed